### PR TITLE
change to new term

### PR DESCRIPTION
**Pull request by morri-son**
## Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)
> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ the sources provide a static file.
 ### `components`
 
 **Optional** The component specification file describing the compoenents to add.
```
Comment (frewilhelm) [inline README.md]:
```suggestion
**Optional** The component specification file describing the components to add.
```

File: README.md [hunk 2]
```diff
@@ -325,13 +325,13 @@ jobs:
             gen/ocm/ctf
 ```
 
-### Using components.yaml
+### Using component-constructor.yaml
```
Comment (frewilhelm) [inline README.md:329]: this seems redundant. The section above already references `component-constructor.yaml`.

**Review by frewilhelm — COMMENTED** [review]
Please consume my review with caution. I have no idea about that syntax :D

**Review by hilmarf — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -24,86 +24,86 @@ Possible actions are
...
-### `directory`
+### `directory` (optional)
 
-**Optional** Will be defaulted to `gen/ocm/component`, if input is required by action.
+Will be defaulted to `gen/ocm/component`, if input is required by action.
 
 The directory to generate the component information
 
-### `component`
+### component (optional)
```
Comment (frewilhelm) [inline README.md]: Why did you remove the `?

File: README.md [hunk 4]
```diff
@@ -24,116 +24,116 @@ Possible actions are
...
-**`directory`**
+### `directory` (optional)
 
-**[OPTIONAL]** Will be defaulted to `gen/ocm/component`, if input is required by action.
+Will be defaulted to `gen/ocm/component`, if input is required by action.
 
 The directory to generate the component information
 
-**`component`**
+### component (optional)
```
Comment (hilmarf) [inline README.md]:
```suggestion
### `component` (optional)
```

File: README.md (hunk 3 above)
Comment (morri-son) [reply README.md]: by fault :-)

Comment (morri-son) [reply README.md]: corrected now

File: README.md (hunk 4 above)
Comment (morri-son) [reply README.md]: good catch. already corrected

**Review by frewilhelm — APPROVED** [review]
